<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveProviderForCreateWidget</class>
 <widget class="QWidget" name="ReceiveProviderForCreateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffffff;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>31</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>31</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>58</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
color:#333333;
font-family:&quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>供应商</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SearchEdit" name="searchEdit" native="true">
        <property name="styleSheet">
         <string notr="true">border: 1px solid #ea8852;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ReceiveProviderListView" name="listView">
     <property name="styleSheet">
      <string notr="true">QListView{
border:none;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>58</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>58</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bottomWidget{
	background-color:#ffffff;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>14</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>24</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="lastPageButton">
           <property name="minimumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
	border-image: url(:/saleImage/btn-up-nomal.png);
}

QPushButton:pressed
{
	border-image: url(:/saleImage/btn-up-pess.png);
}

QPushButton:disabled
{
border-image: url(:/saleImage/btn-up-disable.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>1</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px solid #ededed;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextPageButton">
           <property name="minimumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>87</width>
             <height>37</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/saleImage/btn-down-nomal.png);
}

QPushButton:pressed{
	border-image: url(:/saleImage/btn-down-pess.png);
}

QPushButton:disabled{
	border-image: url(:/saleImage/btn-down-disabled.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pageLabel">
        <property name="styleSheet">
         <string notr="true">font-size:12px;
color:#999999;
font-family:&quot;微软雅黑&quot;;
padding-top:6px;</string>
        </property>
        <property name="text">
         <string>1/1页</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchEdit</class>
   <extends>QWidget</extends>
   <header>SearchEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReceiveProviderListView</class>
   <extends>QListView</extends>
   <header>ReceiveProviderListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
